### DISCUSSION TITLE
One Dark Preset Request

### DISCUSSION THREAD

**Discussion by apuengpreeda** [Akifyss/obsidian-border#158] [Ideas]
Hi, thank you for making this beautiful theme! I enjoy using it!

I know that more presets will be updated soon. May I request a One Dark color scheme ([Reference](https://marketplace.visualstudio.com/items?itemName=zhuangtongfa.Material-theme)) preset as well?

This is my favorite color palette and I hope that you can bring this to the Border theme.

**Comment by Akifyss** [comment]
Added here:
https://github.com/Akifyss/obsidian-border/blob/main/presets/One-dark.json

**Reply by apuengpreeda** [reply]
Thanks for the fast turnaround!
